02-peer-protocol.md

```diff
@@ -263,14 +263,20 @@ The sender MUST wait until the funding transaction has reached
 `minimum-depth` before sending this message.  The sender MUST encode
 the block position of the funding transaction into `channel-id`.  If
 the sender has already received `funding_locked` from the other node,
-it MUST fail the channel if its own `channel-id` does not match the
-received.  The sender MUST set `next-per-commitment-point` to the
+it MAY fail the channel if its own `channel-id` does not match the
+received: otherwise it MUST ignore the `funding_locked` message.
+The sender MAY re-transmit `funding_locked` if the `channel-id` changes.
```

Comment:
I think we can fix this by tying the two together: If either node chooses to ignore, it must re-transmit when it changes.  Because if either node is doing the naive "fail on disagree" it will have failed.